src/index.spec.js

```diff
@@ -2005,7 +2005,17 @@ describe('useDropzone() hook', () => {
 
       fireDrop(dropzone, createDtWithFiles(files))
       await flushPromises(ui, container)
-      expect(onDropSpy).toHaveBeenCalledWith([], files, expect.anything())
+      expect(onDropSpy).toHaveBeenCalledWith([], [
+        {
```

Comment:
I think "prettifying" the `fileRejections` in the tests makes them easier to read. However, it does add quite a few lines to this file. If you don't like the change, lmk and I will put the `expects` all on one line.